spec.emu

```diff
@@ -683,39 +688,87 @@ contributors: Google, Ecma International
           <li><ins>[[rangePatterns]]: </ins></li>
           <ul>
             <li><ins>[[Hour]]:</ins><ul>
-              <li><ins>{[[Source]]: `"startRange"`, [[Pattern]]: `"{day}/{month}/{year}, {hour}:{minute}"`}</ins></li>
-              <li><ins>{[[Source]]: `"shared"`, [[Pattern]]: `" – "`}</ins></li>
-              <li><ins>{[[Source]]: `"endRange"`, [[Pattern]]: `"{hour}:{minute}"`}</ins></li>
+              <li><ins>[[year]]: `"2-digit"`</ins></li>
```

Comment:
Interesting, I wasn't aware of tc39/ecma402#81 and tc39/ecma402#339.

I prefer to keep it lower-case for the moment because updating it would require several updates in 1.1.1 InitializeDateTimeFormat (e.g. Line 28) as well as some parts in 1.3.3 Internal Slots, and I would prefer to minimize the number of changes unrelated to this proposal in order to keep the proposal's PR focused. But I totally agree that we should work on this on a follow-up spec fix.

Given this, I updated the 1.1.6 FormatDateTimePattern and 1.1.10 PartitionDateTimeRangePattern to reference to the property column instead. PTAL